envoy/assets/configuration/spec.yaml

```diff
@@ -38,7 +38,6 @@ files:
           type: string
         example:
           - cluster\.(in|out)\..*
-        default: None
```

Comment:
Why no `default_value: None` here?

Reply:
For the ones that didn't have impact, I removed it. default is not used if it's an optional config option.